uv.lock

```diff
@@ -1191,20 +1191,21 @@ wheels = [
 
 [[package]]
 name = "google-genai"
-version = "1.15.0"
+version = "1.23.0"
```

Comment:
The test failure suggests that with the genai update, the previously recorded cassettes don't match anymore because `post != POST`... Can you try replacing `method: POST` with `method: post` in all the `tests/models/cassettes/test_google/*.yaml` files?